app/src/main/java/com/example/hwr_huschka/klassen/Product.java

```diff
@@ -29,13 +32,16 @@ public Product(int produktID){
 		this.produktID = produktID;
 	}
 
-	public Product(int produktID, String hersteller, String name, String kategorie, double preis, int kcal) {
+	public Product(int produktID, String hersteller, String name, String kategorie, double preis, int kcal, double gesundheitsscore, double umweltscore, String ernaehrungsform) {
```

Comment:
das hier bitte Rückgängig machen, könnte mit bisherigem Code in Konflikt geraten